doc/howtos/ip-address-config.rst

```diff
@@ -0,0 +1,103 @@
+.. _ip-address-config:
+
+Configuring an IP Address in the Ostro |trade| OS
+######################################################
+
+The Ostro Project uses Connection Manager (ConnMan), a Linux daemon and command-line utility, for managing internet connections 
+within embedded devices. 
```

Comment:
I think you're saying that there's a programmatic way to configure IP addresses, and while true, the immediate need was to document how to do this manually.  I'm sure there are plenty of code examples (like this) we need to write about explaining how to do things, that would be appropriate for an application programmer's cookbook with complete examples.